tol_lab_share/message_properties/definitions/sample.py

```diff
@@ -13,6 +13,8 @@
 from tol_lab_share.message_properties.definitions.uuid import Uuid
 from tol_lab_share.message_properties.definitions.dict_input import DictInput
 from tol_lab_share.message_properties.definitions.date_utc import DateUtc
+from tol_lab_share.message_properties.definitions.genome_size import GenomeSize
```

Comment:
Advisory: In here you are importing the definition for each attribute and also the constant for each attribute. Should we be importing the whole thing as a class, object or something else? Don't know much about python so not sure of the options. If you start having other message types it will be quite an overhead if you need to import each attribute. Or are you planning for attributes to be reused in other messages?

Reply:
Yes, the plan is to reuse these attributes if we have more messages in future. It is more verbose but it gives more visibility to what is enabled for a message too. They could be integrated in a template maybe, I think that would be a good refactor but it will depend on the possibility of new different input messages.